cdap-docs/included-applications/source/etl/plugins/transforms/script.rst

```diff
@@ -72,6 +76,15 @@ schema is the same as the input schema.
           {\"name\":\"tax\",\"type\":\"double\"},
           {\"name\":\"total\",\"type\":\"double\"}
         ]
+      }",
+      "lookup": "{
+        \"purchases\":{
+          \"type\":\"DATASET\",
+          \"properties\":{
+            \"dataset_argument1\":\"foo\",
+            \"dataset_argument2\":\"bar\"
```

Comment:
Created https://issues.cask.co/browse/CDAP-4194 and https://issues.cask.co/browse/CDAP-4193